module2/web/build.gradle

```diff
@@ -2,11 +2,14 @@ plugins {
     id 'war'
 }
 
+ext {
+    spring_boot_version = '2.6.3'
+}
+
 dependencies {
     implementation project (':dao')
     implementation project (':service')
-    implementation group: 'org.springframework', name: 'spring-webmvc', version: spring_version
-    implementation group: 'org.springframework', name: 'spring-tx', version: spring_version
-    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.13.0'
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-web', version: spring_boot_version
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-validation', version: spring_boot_version
+    implementation group: 'org.atteo', name: 'evo-inflector', version: '1.3'
```

Comment:
Why you need that dependency?

Reply:
It's used to represent hateoas links name in valid form.